README.md

```diff
@@ -593,6 +593,8 @@ To send a request to the server you can use the following curl command:
 curl -X 'POST' 'http://127.0.0.1:8080/v1/query' -H 'accept: application/json' -H 'Content-Type: application/json' -d '{"query": "write a deployment yaml for the mongodb image"}'
 ```
 
+> You can use `/v1/streaming_query` endpoint (with same parameters) to get the streaming response (SSE/HTTP chunking). By default, it streams text, but you can also yield events as JSONs via additionl `"media_type": "text/plain"` parameter in the payload data.
```

Comment:
You can use **the** `/v1/streaming_query` (with **the** same parameters) (...) JSONs via **additional** ...